src/main/java/nz/cri/gns/NZSHM22/opensha/inversion/MFDManipulation.java

```diff
@@ -67,15 +67,15 @@ public static IncrementalMagFreqDist restrictMFDConstraintMagRange(
         return newMFDConstraints;
     }
 
-    public static UncertainIncrMagFreqDist addMfdUncertainty(IncrementalMagFreqDist mfd, double minimize_below_mag, double minimizeAboveMag, double power) {
+    public static UncertainIncrMagFreqDist addMfdUncertainty(IncrementalMagFreqDist mfd, double minimize_below_mag, double minimizeAboveMag, double power, double uncertaintyScalar) {
         int minMagBin = mfd.getClosestXIndex(minimize_below_mag);
         int maxMagBin = mfd.getClosestXIndex(minimizeAboveMag);
         int firstWeightPowerBin = mfd.getClosestXIndex(FIRST_WEIGHT_POWER_MAG);
         Preconditions.checkArgument(minMagBin <= firstWeightPowerBin,
                 "minMag may not be above the bin of " + FIRST_WEIGHT_POWER_MAG);
         Preconditions.checkArgument( firstWeightPowerBin <= maxMagBin,
                 "maxMag may not be below the bin of " + FIRST_WEIGHT_POWER_MAG);
-        double firstWeightPower = Math.pow(mfd.getY(firstWeightPowerBin), power - 1);
+        double firstWeightPower = Math.pow(mfd.getY(firstWeightPowerBin), power - 1) * (mfd.getY(firstWeightPowerBin) * uncertaintyScalar);
```

Comment:
@chrisbc this is the stray change that I left in that caused the test change

Reply:
Of course - thanks!